app/backend/graphql/team/team.ts

```diff
@@ -29,5 +30,10 @@ export const Team = objectType({
       description: 'List of all customers of the team',
       resolve: (team, _arguments, context) => context.prisma.customer.findMany({ where: { teamId: team.id } }),
     })
+    t.list.field('projects', {
+      type: Project,
+      description: 'List of all projects of the team',
+      resolve: (team, _arguments, context) => context.prisma.project.findMany({ where: { teamId: team.id } }),
```

Comment:
@moncapitaine 
As a team member with the role "Member", am I allowed to see all the projects of the team?

Reply:
I understand your point but find it OK for now.

Reply:
But then we should keep it consistent.
We have a similar query that behaves differently:
https://github.com/progwise/timebook/blob/main/app/backend/graphql/project/queries/projectsQueryField.ts